openverse_api/catalog/api/views/media_views.py

```diff
@@ -105,7 +105,7 @@ def related(self, request, identifier=None, *_, **__):
             )
             self.paginator.result_count = num_results
             self.paginator.page_count = 1
-            self.paginator.page_size = num_results
+            self.paginator.page_size = 10
```

Comment:
We should probably add a note here saying that this used to be `num_results` and may need to be set back!